### ISSUE TITLE
Creation of collection and stored procedure upon every instantiation?

### ISSUE THREAD

**Issue by deyanp** [Dzoukr/CosmoStore#23]
Is it possible to skip the collection/stored procedure check/creation? Sounds like at least 2 roundtrips which are not needed for subsequent calls, or am I missing something?

**Comment by Dzoukr** [comment]
Hi @deyanp,

currently it is not possible, but it happens only when you instantiate EventStore, which should happen only once per application. It is necessary to be sure that collection & stored procedure exist (in correct version).

**Comment by deyanp** [comment]
Hi @Dzoukr,

I intend to use this in Azure Function (serverless), and even though the function is kept around for some time, it will be also very often the case that a new instance of the function is created on regular basis. Any startup code or IO is going to have an impact ...

**Comment by Dzoukr** [comment]
Fixed in v2.1.0

**Comment by deyanp** [comment]
Thanks a lot, @Dzoukr!!!
